scenarios/02_Welcome_to_Wesnoth.cfg

```diff
@@ -796,28 +788,28 @@ You can easily identify heroic units by the special overlay to the right of thei
     name=die
 
         [filter]
-            id=Sir_Kellan
+            id=Sir_Elec
         [/filter]
 
         [switch]
-		variable=second_unit.side
-		[case]
-			value=1
-			
-			[message]
-				speaker=Bazur
-				message=_ "We did it! We took the city and killed the rebel chief, queen!"
-			[/message]
+        variable=second_unit.side
+        [case]
+            value=1
+            
+            [message]
+                speaker=Bazur
+                message=_ "Hah, we did it! Hey queen, we took the city and killed the rebel chief!"
```

Comment:
Ok, I'll change it to "Queen".  I'm not sure if it's best to capitalize or not, but since you capitalize Queen in most other places we should capitalize it here too.